chartdata.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+function newElement(elType, elAttribute, elAttributeName, elParentId, elText){
+  var el = document.createElement(elType);
+  el.setAttribute(elAttribute, elAttributeName);
+  el.textContent = elText;
+  var parentEl = document.getElementById(elParentId);
+  parentEl.appendChild(el);
+}
+
+var imgs = JSON.parse(localStorage.imgs);
```

Comment:
You do not have a localStorage.imgs stored inside local storage. Did  you mean .product? This should fix your issue with the chart.